packages/govuk-frontend/src/govuk/components/footer/_index.scss

```diff
@@ -71,16 +71,27 @@
     text-wrap: balance;
   }
 
-  .govuk-footer__copyright-logo {
-    display: inline-block;
-    min-width: $govuk-footer-crest-image-width;
-    padding-top: ($govuk-footer-crest-image-height + govuk-spacing(2));
+  .govuk-footer__copyright-logo::before {
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: ($govuk-footer-crest-image-height + govuk-spacing(2));
     background-image: govuk-image-url("govuk-crest.svg");
     background-repeat: no-repeat;
     background-position: 50% 0%;
     background-size: $govuk-footer-crest-image-width $govuk-footer-crest-image-height;
     text-align: center;
     white-space: nowrap;
+    filter: brightness(0) saturate(100%) invert(3%) sepia(5%) saturate(752%) hue-rotate(132deg) brightness(94%)
+      contrast(97%);
```

Comment:
@patrickpatrickpatrick Given the colour of the crest is in our control, we can work with it to simplify the filter, rather than going for a generic filter that would work for any colour of the crest.

The initial colour is `#6E777A`, which is `hsl(195, 5%, 45%)`. The target colour is `#0b0c0c`, which is `hsl(0, 4%, 5%)`. This means we need to:
- reduce the hue of 195deg
- scale the brightness from 45% to 5%

```suggestion
    filter: hue-rotate(-195deg) brightness(calc(5 / 45));
```

Given the darkness we're operating at, I doubt anyone will notice a difference if we only reduce the brightness to 5% with:

```suggestion
	filter: brightness(calc(5 / 45));
```

That's something we should run past @CharlotteDowns though.

Reply:
Made screenshots of the 3 filters on a very zoomed in logo (so it's easier to perceive colour difference if any), if it helps deciding:

1. Original filter from the PR
<img width="1511" alt="Screenshot 2025-03-25 at 19 35 12" src="https://github.com/user-attachments/assets/76d2789d-f839-48be-8dbd-e307f596baed" />

2. `hue-rotate` + `brightness`
<img width="1512" alt="Screenshot 2025-03-25 at 19 35 32" src="https://github.com/user-attachments/assets/4c432302-8bc7-472b-821a-67285162e195" />

3. `brightness` only
<img width="1512" alt="Screenshot 2025-03-25 at 19 35 45" src="https://github.com/user-attachments/assets/28917af0-8961-4e49-9260-43705f322152" />

Reply:
I'm happy to approve the lean version `filter: brightness(calc(5 / 45));` based on a screenshot comparison I've made and feedback from the team.

![Three crown copyright logos, one using #0b0c0c in the svg colour, the others applying filters on top of the existing grey colour](https://github.com/user-attachments/assets/0e19f0cb-2eab-4f18-96e5-a354061a407e)